pkg/chain_iterator/block_batch_iterator.go

```diff
@@ -174,9 +175,16 @@ func (i *BlockBatchIterator) iter() (err error) {
 	if i.endHeight != nil {
 		destHeight = *i.endHeight
 	} else {
-		if destHeight, err = i.client.BlockNumber(i.ctx); err != nil {
+		destHeight, err = i.client.BlockNumber(i.ctx)
+		if err != nil {
 			return err
 		}
+		if destHeight > DefaultBlockConfirmations {
```

Comment:
Looks like the confirmations are not configurable at all, so its always `0`?

Reply:
fix it later